docs/UserGuide.adoc

```diff
@@ -161,51 +202,120 @@ Examples:
+Format: `stats`
+
+
+=== Todo Mode
+
+In Todo Mode, manipulating of eateries which user had yet to visit will be available. +
+
+==== Toggling between main and todo modes: `mode`
+
+Toggles between main and todo modes. +
+Format: `mode`
+
+==== Adding an eatery: `add`
+
+Adds an eatery to the todo list. +
+Format: `add \n [name of eatery] \a [address] {\t [tags, space-delimited]}`
 
 Examples:
 
-* `import \f /home/user/eatme-backup-john.txt`
+* `add \n Two Chefs Eating Place \a 116 Commonwealth Cres, #01-129`
+* `add \n Two Chefs Eating Place \a 116 Commonwealth Cres, #01-129 \t good-for-groups non-aircon`
 
-==== Exporting eateries: `export`
+==== Saving a todo eatery to the main eatery list : `save`
+
+Remove a todo eatery from the todo list and provides a quick way to add it to the main list of eateries. +
+Format: `save [id]` (saves the eatery to the main list)
+
+Examples:
+
+* `save 2`
+
+==== Finding an eatery: `find`
 
 *Coming in v1.4*
 
-Exports eateries to a text file. +
-Format: `export \f [filename]`
+Finds an eatery based on name or tag. +
+Format: `find \n [name of eatery]` or `find \t [tag]`
 
 Examples:
 
-* `export \f /home/user/eatme-backup-john.txt`
+* `find \n Two Chefs Eating Place`
+* `find \t Chinese`
 
-=== Visualisation
-==== Showing overall statistics: `stats`
+==== Changing address of an eatery: `changeaddress`
 
 *Coming in v1.4*
 
-Presents a visual overview of the user’s eateries and reviews. +
-Format: `stats`
+Changes the address of an eatery. +
+Format: `changeaddress [id] \a [new address]`
 
-=== Todo
-==== Toggling between main and todo modes: `mode`
+Examples:
 
-Toggles between main and todo modes. +
-Format: `mode`
+* `changeaddress 2 \a 116 Commonwealth Cres, #01-130`
 
-==== Saving a todo eatery to the main eatery list : `save`
+==== Marking an eatery as closed: `close`
 
-Remove a todo eatery from the todo list and provides a quick way to add it to the main list of eateries. +
-Format: `save [id]` (saves the eatery to the main list)
+Closes an eatery if the eatery is no longer there. The eatery will still be listed, but will be grayed out to inform you that the eatery no longer exists. In the case that you accidentally closed the wrong eatery or the eatery reopens, typing `reopen` does the opposite of `close`. +
+Format: `close [id]`
 
 Examples:
 
-* `save 2`
+* `close 2`
+
+==== Reopening a closed eatery: `reopen`
+
+Reopens a previously closed eatery. +
+Format: `reopen [id]`
+
+Examples:
+
+* `reopen 2`
+
+==== Tagging an eatery: `addtag`
+
+Adds tags to an eatery. +
+Format: `addtag [id] \t [tags, space-delimited]`
+
+Examples:
+
+* `addtag 2 \t hawker good-for-sharing`
+
+==== Removing tags from an eatery: `removetag`
+
+Removes tags from an eatery. +
+Format: `removetag [id] \t [tags, space-delimited]`
+
+Examples:
+
+* `removetag 2 \t hawker`
+
+==== Viewing an eatery: `show`
+
+Shows an eatery with all its details - Address, category, tags, reviews, external reviews and a map of its location. +
```

Comment:
Shows an eatery with all its details i.e. address, category, tags, reviews, external reviews (from Google), and a map of its location.